src/Translator.php

```diff
@@ -148,13 +148,17 @@ private function loadTranslationsFromFile(string $filename): void
             $translations = $stream->readint($unpack, 16);
 
             /* get original and translations tables */
-            $tableOriginals = $stream->readintarray($unpack, $originals, $total * 2);
-            $tableTranslations = $stream->readintarray($unpack, $translations, $total * 2);
+            $totalTimeTwo = (int) ($total * 2);// Fix for issue #36 on ARM
```

Comment:
Do you want to call this totalTimesTwo? Usually "times" with an "s" is used for multiplication so that might be better than "time".

Otherwise looks good to me. Nice work tracking this down.

Reply:
Sure, good catch